README.md

```diff
@@ -1,15 +1,13 @@
 [![Build Status](https://semaphoreci.com/api/v1/calico/libcalico-go/branches/master/badge.svg)](https://semaphoreci.com/calico/libcalico-go)[![Slack Status](https://slack.projectcalico.org/badge.svg)](https://slack.projectcalico.org) [![IRC Channel](https://img.shields.io/badge/irc-%23calico-blue.svg)](https://kiwiirc.com/client/irc.freenode.net/#calico)
 
 # libcalico-go
-This repositiory contains Calico's Go components:
+This repositiory contains Calico's shared Go library, `libcalico`, which can be imported as 
+`"github.com/projectcalico/libcalico-go/lib"`
```

Comment:
Is that really true and/or meaningful?  Looking at the imports in Felix, I see:

    ./felix/tagindex/tag_index.go:20:	"github.com/projectcalico/libcalico-go/lib/backend/api"
    ./felix/tagindex/tag_index.go:21:	"github.com/projectcalico/libcalico-go/lib/backend/model"
    ./felix/dispatcher/dispatcher.go:19:	"github.com/projectcalico/libcalico-go/lib/backend/api"
    ./felix/dispatcher/dispatcher.go:20:	"github.com/projectcalico/libcalico-go/lib/backend/model"
    ./felix/config/config_suite_test.go:21:	"github.com/projectcalico/libcalico-go/lib/testutils"
    ./felix/config/config_params.go:21:	"github.com/projectcalico/libcalico-go/lib/api"
    ./felix/config/config_params.go:22:	"github.com/projectcalico/libcalico-go/lib/backend/etcd"
    ./felix/config/config_params.go:23:	"github.com/projectcalico/libcalico-go/lib/client"
    ./felix/ip/ip_addr.go:24:	calinet "github.com/projectcalico/libcalico-go/lib/net"
    etc.